### PR TITLE
fix(input-amount): normalize added for currency label

### PR DESCRIPTION
**Pull request by demeteren**
I created a method that would normalize currency label. It fixes a Turkish currency problem, that would show TRY for locale TR which is unknown for Turkey. It is an override method.

### CODE REVIEW

**Comment by demeteren** [comment]
@gerjanvangeest hi, not sure if you recieve notification after a pull request is made. CI changed since my last contribution. Not sure about this one :)
Thanks

File: packages/input-amount/src/formatters.js [hunk 1]
```diff
@@ -23,3 +23,10 @@ export function formatAmount(modelValue, givenOptions) {
 
   return formatNumber(modelValue, options);
 }
+
+export function normalizeCurrencyLabel(currency, locale) {
```
Comment (tlouisse) [inline packages/input-amount/src/formatters.js]: Good fix. Maybe it's an idea to put this function in the `localize` package as a file inside `src/number`? There we keep all corrections on top of `Intl.Numberformat`

File: packages/input-amount/src/LionInputAmount.js [hunk 2]
```diff
@@ -127,4 +128,12 @@ export class LionInputAmount extends LocalizeMixin(LionInput) {
     // sufficient, it should also contain the abbreviation.
     this._currencyDisplayNode.setAttribute('aria-label', getCurrencyName(this.currency));
   }
+
+  __CurrencyLabel() {
```
Comment (tlouisse) [inline packages/input-amount/src/LionInputAmount.js]: Maybe make this `get __currencyLabel() `?

Comment (demeteren) [reply packages/input-amount/src/LionInputAmount.js]: Thanks @tlouisse . Renamed the method name.

File: packages/input-amount/src/formatters.js (hunk 1 above)
Comment (demeteren) [reply packages/input-amount/src/formatters.js]: @tlouisse Thanks, exactly did that. Is it good now ?

File: packages/input-amount/src/LionInputAmount.js [hunk 3]
```diff
@@ -127,4 +128,12 @@ export class LionInputAmount extends LocalizeMixin(LionInput) {
     // sufficient, it should also contain the abbreviation.
     this._currencyDisplayNode.setAttribute('aria-label', getCurrencyName(this.currency));
   }
+
+  __getCurrencyLabel() {
```
Comment (jorenbroekema) [inline packages/input-amount/src/LionInputAmount.js]:
```suggestion
  get __currencyLabel() {
```

I think this is what @tlouisse  meant

**Comment by jorenbroekema** [comment]
Thanks for your contribution 🤗

**Pull request merged**